### ISSUE TITLE
Train and validation loss

### ISSUE THREAD

**Issue by mehadi-trackrep** [SeanNaren/deepspeech.pytorch#379]
In train.py file, is  'loss.val'  for validation loss? So, which is for training loss?

**Comment by ronybanerjee93** [comment]
loss.val suggests the 'val'ue of loss.

**Comment by mehadi-trackrep** [comment]
Okay. But, I want to get the train and validation loss separately.
Would you tell me the functions that are relevant here?

>
